fastly/block_fastly_service_v1_logging_kafka_test.go

```diff
@@ -53,6 +58,11 @@ func TestResourceFastlyFlattenKafka(t *testing.T) {
 					"format":             `%a %l %u %t %m %U%q %H %>s %b %T`,
 					"placement":          "none",
 					"format_version":     uint(2),
+					"parse_log_keyvals":  true,
+					"max_batch_size":     uint(12345),
```

Comment:
Good question. This is necessary because the `FormatVersion` and `RequestMaxBytes` fields in the`Kafka` type in `go-fastly` are `uint` types and they are implicitly converted to `uint` when the struct is created. The casting is needed at the point of this comment because there is no type expectation and the values will default to `int` types and this causes the test comparison between `remote` and `local` to fail.